src/masternode/masternode-sync.h

```diff
@@ -29,6 +31,8 @@ extern CMasternodeSync masternodeSync;
 class CMasternodeSync
 {
 private:
+    CCriticalSection cs;
```

Comment:
Let's maybe
```suggestion
    mutable CCriticalSection cs;
```
and bring back `const`s you dropped?